test/views/staging-view.test.js

```diff
@@ -355,14 +355,17 @@ describe('StagingView', function() {
         });
         document.body.appendChild(view.element);
 
-        const getPendingFilePatchItem = sinon.stub(view, 'getPendingFilePatchItem').returns(false);
+        const getPanesWithStalePendingFilePatchItem = sinon.stub(view, 'getPanesWithStalePendingFilePatchItem').returns([]);
         await view.selectNext();
         assert.isFalse(showFilePatchItem.called);
 
-        getPendingFilePatchItem.returns(true);
+        getPanesWithStalePendingFilePatchItem.returns(['item1', 'item2']);
         await view.selectPrevious();
+        assert.isTrue(showFilePatchItem.calledTwice);
```

Comment:
Since this is `.calledTwice`, shouldn't there be two `assert.isTrue(showFilePatchItem.calledWith(...))` calls below it to assert that they're both correct? Or are the other calls too annoying to assert specifically? 🤔 

Same for the next stanza there 👇